pom.xml

```diff
@@ -107,6 +107,12 @@
                 <version>5.6.4</version>
                 <scope>provided</scope>
             </dependency>
+            <dependency>
```

Comment:
Note, that it's a version from AEM 6.1 and does not seems to be publicly available.